### PR TITLE
Add cloud_run_revision MR to stackdriver filter

### PR DESCRIPTION
**Pull request by bochengchu**

### CODE REVIEW

**Review by kyessenov — COMMENTED** [review]
LGTM

File: extensions/stackdriver/common/constants.h [hunk 1]
```diff
@@ -104,8 +105,11 @@ constexpr char kPodNameLabel[] = "pod_name";
 constexpr char kContainerNameLabel[] = "container_name";
 constexpr char kGCEInstanceIDLabel[] = "instance_id";
 constexpr char kZoneLabel[] = "zone";
-constexpr char kNamespaceLabel[] = "namespace"; // used for generic_node
-constexpr char kNodeIDLabel[] = "node_id";      // used for generic_node
+constexpr char kNamespaceLabel[] = "namespace";                  // used for generic_node
+constexpr char kNodeIDLabel[] = "node_id";                       // used for generic_node
+constexpr char kServiceNameLabel[] = "service_name";             // used for cloud_run_revision
```
Comment (lei-tang) [inline extensions/stackdriver/common/constants.h:110]: kServiceNameLabel -> kGCPCRServiceNameLabel

File: extensions/stackdriver/common/constants.h [hunk 2]
```diff
@@ -104,8 +105,11 @@ constexpr char kPodNameLabel[] = "pod_name";
...
+constexpr char kRevisionNameLabel[] = "revision_name";           // used for cloud_run_revision
```
Comment (lei-tang) [inline extensions/stackdriver/common/constants.h:111]: kRevisionNameLabel -> kGCPCRRevisionNameLabel

File: extensions/stackdriver/common/constants.h [hunk 3]
```diff
@@ -104,8 +105,11 @@ constexpr char kPodNameLabel[] = "pod_name";
...
+constexpr char kConfigurationNameLabel[] = "configuration_name"; // used for cloud_run_revision
```
Comment (lei-tang) [inline extensions/stackdriver/common/constants.h:112]: kConfigurationNameLabel -> kGCPCRConfigurationNameLabel

File: extensions/stackdriver/stackdriver.cc [hunk 4]
```diff
@@ -221,7 +221,8 @@ flatbuffers::DetachedBuffer getLocalNodeMetadata() {
       *flatbuffers::GetRoot<::Wasm::Common::FlatNode>(local_node_info.data()), &node);
   const auto mesh_id_it = node.fields().find("MESH_ID");
   if (mesh_id_it != node.fields().end() && !mesh_id_it->second.string_value().empty() &&
-      absl::StartsWith(mesh_id_it->second.string_value(), "proj-")) {
+      (absl::StartsWith(mesh_id_it->second.string_value(), "proj-") ||
+       absl::StartsWith(mesh_id_it->second.string_value(), "projects/"))) {
```
Comment (lei-tang) [inline extensions/stackdriver/stackdriver.cc:225]: Can you add a comment to explain why this change is needed?

File: extensions/stackdriver/common/constants.h (hunk 1 above)
Comment (bochengchu) [reply extensions/stackdriver/common/constants.h]: Done

File: extensions/stackdriver/common/constants.h (hunk 2 above)
Comment (bochengchu) [reply extensions/stackdriver/common/constants.h]: Done

File: extensions/stackdriver/common/constants.h (hunk 3 above)
Comment (bochengchu) [reply extensions/stackdriver/common/constants.h]: Done

File: extensions/stackdriver/stackdriver.cc (hunk 4 above)
Comment (bochengchu) [reply extensions/stackdriver/stackdriver.cc:227]: Added in the comment of getLocalNodeMetadata().

**Review by lei-tang — APPROVED** [review]
